internal/api/apiservice/rebuild.go

```diff
@@ -200,6 +202,10 @@ func getStrategy(ctx context.Context, deps *RebuildPackageDeps, t rebuild.Target
 }
 
 func buildAndAttest(ctx context.Context, deps *RebuildPackageDeps, mux rebuild.RegistryMux, a verifier.Attestor, t rebuild.Target, strategy rebuild.Strategy, entry *repoEntry, useProxy bool, useSyscallMonitor bool) (err error) {
+	debugStore, err := deps.DebugStoreBuilder(ctx)
+	if err != nil {
+		return errors.Wrap(err, "creating debug store")
+	}
 	id := uuid.New().String()
```

Comment:
can be a separate PR but I think this ID should be recorded in Firestore to avoid needing to access GCS to know how to read the logs. Alternatively, as mentioned in the TUI PR, I think it might be reasonable to copy the logs, no matter how large, to the debug store after execution.

Reply:
Already mailed in #194. Just FYI this uuid is not the uuid needed for the logs. The logs are written by the GCB service using the GCB generated BuildID, whereas this uuid is our oblivious uuid. Related though, because #194 adds both ids to the firestore record.